platform/view/services/comm/host/rest/websocket/stream_test.go

```diff
@@ -104,19 +105,24 @@ func TestReader(t *testing.T) {
 	for _, message := range input {
 		assert.NoError(t, conn.ReadValue(message))
 	}
+	wg := sync.WaitGroup{}
```

Comment:
why the changes in this file? It seems unrelated to the rest.

Reply:
This also caused the tests to fail when run with `-race`.